src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +292,31 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                try:
+                    should_get_codecov_line_coverage = (
+                        features.has(
+                            "organizations:codecov-stacktrace-integration",
+                            project.organization,
+                            actor=request.user,
+                        )
+                        and project.organization.flags.codecov_access
+                    )
+                    if should_get_codecov_line_coverage:
+                        (
+                            result["lineCoverage"],
+                            result["codecovStatusCode"],
+                        ) = self.get_codecov_line_coverage(
+                            current_config["config"]["repoName"],
+                            current_config["config"]["provider"]["key"],
+                            current_config["config"]["defaultBranch"],
+                            result["sourceUrl"],
```

Comment:
Could you return `formattedPath` as `source_path` in `get_link`?
https://github.com/getsentry/sentry/blob/b763ac19e1f40b5ea57dda177f0f13712f220b83/src/sentry/api/endpoints/project_stacktrace_link.py#L33 

You can then just pass `current_config["outcome"]["source_path"]` and avoid making transformations inside of `get_codecov_line_coverage`

Reply:
Thank you, this suggestion is very helpful!